### PR TITLE
Penalize ARC for long reviews

### PR DESCRIPTION
**Pull request by g4s8**
#1028 - add negative points to `ARC` when closing old pull-request in GIthub

### CODE REVIEW

**Comment by 0crat** [comment]
Job [#1703](https://github.com/zerocracy/farm/issues/1703) is now in scope, role is `REV`

**Comment by 0crat** [comment]
This pull request [#1703](https://github.com/zerocracy/farm/issues/1703) is assigned to @krzyk[/z](https://www.0crat.com/u/krzyk), here is [why](https://www.0crat.com/footprint/C3NDPUA8L/2611410295); the budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @g4s8[/z](https://www.0crat.com/u/g4s8) (the architect) right in this ticket; if you decide that this PR should not be accepted ever, also inform the architect; this [blog post](http://www.yegor256.com/2015/02/09/serious-code-reviewer.html) will help you understand what is expected from a code reviewer; there will be a monetary reward for this job

File: src/main/groovy/com/zerocracy/stk/pm/cost/penalize_architect_for_long_pull_request.groovy [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+        .param('reason', new Par('Code review was too long, see §55').say())
+        .postTo(new ClaimsOf(farm))
+    }
+    String logins = String.join(', ', new Mapped({ "@${it}" }, arcs))
+    claim.copy()
+      .type('Notify job')
+      .token("job;${job}")
+      .param(
+      'message',
+      new Par("Code review was too long (%d days), architects ${logins} was penalized, see §36")
```
Comment (krzyk) [inline src/main/groovy/com/zerocracy/stk/pm/cost/penalize_architect_for_long_pull_request.groovy]: @g4s8 `was` or `were` depending on the number of arcs, so maybe put the architects in `()` and use `were`?

Comment (krzyk) [reply src/main/groovy/com/zerocracy/stk/pm/cost/penalize_architect_for_long_pull_request.groovy]: @g4s8 should it be paragraph 55?

File: src/test/resources/com/zerocracy/bundles/penalize_arc_for_long_pr/_after.groovy [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+import com.jcabi.xml.XML
+import com.zerocracy.Farm
+import com.zerocracy.Project
+import com.zerocracy.pmo.Awards
+import org.hamcrest.MatcherAssert
+import org.hamcrest.core.IsEqual
+
+def exec(Project project, XML xml) {
+  Farm farm = binding.variables.farm
+  Awards awards = new Awards(farm, 'arc').bootstrap()
```
Comment (krzyk) [inline src/test/resources/com/zerocracy/bundles/penalize_arc_for_long_pr/_after.groovy]: @g4s8 this can be inlined

**Comment by krzyk** [comment]
@g4s8 you also need to fix other tests that check for ARC points, e.g. awards_points_to_arc_for_pull_request fails right now

**Comment by krzyk** [comment]
@g4s8 thanks

**Comment by g4s8** [comment]
@rultor merge

**Comment by g4s8** [comment]
@rultor merge

**Comment by 0crat** [comment]
The job [#1703](https://github.com/zerocracy/farm/issues/1703) is now out of scope

**Comment by ypshenychka** [comment]
@krzyk According to our [QA Rules](http://www.zerocracy.com/policy.html#42):
>The code reviewer found at least three problems in the code.
Comments were mostly about design problems, not cosmetic issues.

Only [one](https://github.com/zerocracy/farm/pull/1703#issuecomment-422689799) major issue was found during code review. 
Please confirm that you'll try to find at least three major problems while future reviews.

**Comment by krzyk** [comment]
@yegor256 I confirm

**Comment by ypshenychka** [comment]
@krzyk thanks

**Comment by ypshenychka** [comment]
@0crat quality acceptable

**Comment by 0crat** [comment]
Order was finished, quality is "acceptable": +15 point(s) just awarded to @krzyk[/z](https://www.0crat.com/u/krzyk)

**Comment by 0crat** [comment]
Quality review completed: +8 point(s) just awarded to @ypshenychka[/z](https://www.0crat.com/u/ypshenychka)

**Pull request merged**